src/dra/4A538.c

```diff
@@ -615,13 +615,14 @@ void LoadEquipIcon(s32 equipIcon, s32 palette, s32 index) {
 
     if (D_80137538[index] != palette) {
         for (i = 0; i < 16; i++) {
-            if (D_800705CC) { // FAKE
+            if (g_Clut[0x1D00]) { // FAKE
```

Comment:
```suggestion
            if (g_Clut[0x1D00]) {
```
let's remove the `FAKE` label

Reply:
This was bugging me too actually, so I was looking into the PSP side about how to solve. I have worked out how to get rid of this FAKE and have updated the PR